src/controllers/crop_controller_old.py

```diff
@@ -0,0 +1,173 @@
+import random
+import bpy
```

Comment:
If this old version of crop controller isn't being used we can remove it completely. If we need a reference we can check the commit history.

Reply:
Yes I forgot to remove it, good spot